logr/logr.go

```diff
@@ -0,0 +1,90 @@
+// Package logr contains convenience functions for accessing a http.Request's logger.
+package logr
+
+import (
+	"context"
+	"github.com/authenticvision/util-go/logutil"
+	"log/slog"
+	"net/http"
+)
+
+func Enabled(r *http.Request, level slog.Level) bool {
+	ctx := r.Context()
+	log := logutil.FromContext(ctx)
+	return log.Enabled(ctx, level)
+}
```

Comment:
This functionality feels pretty redundant/too mayn ways to do one thing and I'd prefer just doing `log := logutil.FromContext(ctx)` (maybe even with a `Scope`) whereever a logger is needed instead at the top of the function

Reply:
[Still not sure whether I like it without](https://git.avdev.at/dev/helios-all/-/commit/163080d511404cdc2a4fb2cf156722a8c690496f), but I dropped the logr commit from this PR. I managed to avoid most extra lines (at the cost of an ugly replacement of `log`) after refactoring that function yet again in a previous commit.

The `logr` variant would not construct a logger when none was needed before, but that's a micro optimization anyway.

Reply:
FWIW, for something like `logutil.FromContext((*r).Context()).Info("user session created")` I'd still split out the the logger into a variable at the top of the function. Better one line more and better to read than a line less